src/app/utilities/recipe.utility.ts

```diff
@@ -487,7 +487,7 @@ export class RecipeUtility {
       netProductionOnly,
       data,
     );
-    this.adjustCost(recipeIds, recipeR, recipesState, cost);
+    this.adjustCost(recipeIds, recipeR, recipesState, cost, data);
```

Comment:
Do you want to comment out the code changes in this file so that the tests can pass, we can merge, and I can uncomment the code and work on updating the tests?

Reply:
i didn't look at the tests at first in case you'd want it implemented differently anyway. i've updated the tests.

i'm pretty sure this test will have to be updated at the same time the data sets are updated, though:
https://github.com/escalonn/factoriolab/blob/sea-area-costs/src/app/utilities/recipe.utility.spec.ts#L833